common-libs/internal-api-clients/src/services/KeyStorageApiService.ts

```diff
@@ -10,7 +10,7 @@ type GetSignedCredentialOptions = {
   // For remote IssuerApiService (used by networkMember.verifyCredentialOfferResponseToken)
   env?: Env
   issuerUrl?: string
-  accessApiKey?: string
+  accessApiKey: string
```

Comment:
As I remember, any call to this service require apiKey, so it is not breaking change